MIGRATION.md

```diff
@@ -1,3 +1,10 @@
+# Riptide 4.0 Migration Guide
+
+## Breaking Changes
+
+Since Spring 5, `RestTemplate` and `AsyncRestTemplate` are deprecated in favor of `WebClient`.
```

Comment:
Only `AsyncRestTemplate` deprecated in favour `WebClient`

Reply:
Sorry. The relevant classes are restored.